### ISSUE TITLE
distance property doesn't exist in PlotSeriesDataLabelsOptions

### ISSUE THREAD

**Issue by a688** [highcharts/highcharts#20673]
#### Expected behaviour
Copy https://www.highcharts.com/demo/highcharts/pie-drilldown into an Angular app and run without any issues

#### Actual behaviour
distance property (set to -30% in the demo) doesn't exist on the TypeScript object PlotSeriesDataLabelsOptions

#### Live demo with steps to reproduce
N/A (do you have angular demos of each chart type?)

#### Product version
Highcharts 11.3.0

#### Affected browser(s)
N/A

**Comment by bm64** [comment]
Hello @a688, and thanks for reporting the issue.

As a workaround you can extend `PlotSeriesDataLabelsOptions` interface with `distance` property: https://stackblitz.com/edit/highcharts-angular-property-xxx-doesnt-exist-oc7g6p?file=src%2Fapp%2Fapp.component.ts

**Comment by damisparks** [comment]
Also, I can confirm the issue. @bm64 I appreciate the workaround.
My workaround: 

```ts
/**
 * Augment highcarts types to include the missing properties
 * see: https://github.com/highcharts/highcharts/issues/20673
 */
declare module 'highcharts' {
  interface PlotSeriesOptions {
    borderRadius?: number; // Add missing borderRadius property
  }
  interface PlotSeriesDataLabelsOptions {
    distance?: number; // Add missing distance property
  }
}
```

**Comment by Eskils** [comment]
Hey @a688,

Thank you for being patient with this issue.

It seems to me that the demo that you copied is “logically wrong”  since setting `distance` in **series** plotOptions affects all series and not all series has the `distance` option.

What you should do instead is set the `distance` property in the **pie** plot options. There was however a bug that caused Highcharts to crash in some cases when using the pie plotOptions.

This issue has now been resolved on the master branch, and will make it into the next version of Highcharts.
The demos will then be updated.

---

**In short**: In the next version of Highcharts, this should work in Typescript:

```js
plotOptions: {
    pie: {              // Change this from `series` to `pie`
         dataLabels: [{
            enabled: true,
            distance: 20
        }, {
            enabled: true,
            distance: '-30%',
            format: '{point.percentage:.1f}%',
        }]
    }
}
```

**Comment by AndrzejBuleczka** [comment]
Hi @Eskils,
If this demo with the v12.2.0 doesn't throw typescript errors, does it mean that the issue is now resolved?
https://stackblitz.com/edit/highcharts-angular-line-wkb6wrm5?file=src%2Fapp%2Fapp.component.ts

**Comment by Eskils** [comment]
Yes, this issue is resolved.
